.github/workflows/ci.yml

```
@@ -312,9 +312,11 @@ jobs:
           node-version: "24.18.1"
           cache: yarn
       - run: yarn install --frozen-lockfile
-      - run: npx playwright install --with-deps chromium
       - name: Build production Worker
         run: yarn build
+      - name: Capture Nuxt build ID
+        id: nuxt_build
+        run: echo "id=$(node -p \"JSON.parse(require('fs').readFileSync('.output/public/_nuxt/builds/latest.json', 'utf8')).id\")" >> "$GITHUB_OUTPUT"
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1009:info:1:6: The mentioned syntax error was in this double quoted string

(shellcheck)